probablefutures-py/probablefutures/output.py

```diff
@@ -0,0 +1,16 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Input:
```

Comment:
Maybe you meant "Output"? 😁

Reply:
This is fixed those these classes aren't really available for use in the API yet.  But hopefully soon